lib/Munin/Node/Config.pm

```diff
@@ -340,6 +343,15 @@ sub _apply_wildcard_to_service {
 }
 
 
+sub _convert_bool {
```

Comment:
In `Munin::Common::Config` there is `_parse_bool` - maybe you could use it here?
(it accepts a broader range of string inputs)

Reply:
I didn't know about this function, but i do not like it. It's somehow confusing to me.
More acceptable values like (`on`, `off`, `1`, `0`) are feasible.

Reply:
Maybe you see a good way to improve the function?
I would like to prevent code duplication for boolean parsing, if possible.

Reply:
> More acceptable values like (on, off, 1, 0) are feasible.

We can *add* values, but removing them is always very tricky.